BazelExtensions/workspace.bzl

```diff
@@ -205,10 +216,12 @@ def new_pod_repository(name,
 
          url: the url of this repo
 
-         podspec_url: the podspec url. By default, we will look in the root of
-         the repository, and read a .podspec file. This requires having
-         CocoaPods installed on build nodes. If a JSON podspec is provided here,
-         then it is not required to run CocoaPods.
+         podspec_url: an override podspec file. Can be either a URL or a Bazel
```

Comment:
did you mean to add a separate doc for `podspec_file` instead of `podspec_url`?

Reply:
Nope, this has been changed such that the public API is only `podspec_url`. The macro will then fill in `podspec_url` or `podspec_file` on the underlying rule as is appropriate.